packages/story-editor/src/components/form/tags/reducer.js

```diff
@@ -137,10 +138,10 @@ function reducer(state, action) {
       };
     }
 
-    case ACTIONS.RESET_OFFSET: {
+    case ACTIONS.RESET_VALUE: {
```

Comment:
I don't see you calling this anymore with the current setup.

The only bug I see resulting from this is that we do want to reset the offset whenever we blur the input and we're not submitting a value

Reply:
so `RESET_OFFSET` isn't part of the `ACTIONS` outlined that are exported and available, so it wasn't ever actually triggering this case when it's "called" in the tag input.

Reply:
ooo. I think it was getting triggered, it was just mapping to `undefined`. Cus I see the functionality working on main, and not on this branch. Good catch tho

Reply:
I think we still need that behavior in there regardless tho

Reply:
weird that it was working in main since there's no diff here removing it? I'll restore it!